ci/validate/decorators.py

```diff
@@ -145,6 +146,46 @@ def validate(*args, **kwargs):
     return validate
 
 
+def required_vdisk(func):
+    """
+    Validate if vdisk exists on a vpool
+
+    :param func: function
+    :type func: Function
+    """
+
+    def validate(*args, **kwargs):
+        # check if preset exists or not on existing alba backend
+        if kwargs['vpool_name'] and kwargs['vdisk_name']:
```

Comment:
Could result in a KeyError.
You should catch this one and log a appriorate error (they were not specified in the method')